coordinator/ethereum/test-utils/src/main/kotlin/net/consensys/zkevm/ethereum/AccountManager.kt

```diff
@@ -231,3 +231,32 @@ object L2AccountManager : AccountManager by WhaleBasedAccountManager(
   genesisFile = getPathTo(System.getProperty("L2_GENESIS", "docker/config/linea-local-dev-genesis.json")),
   log = LogManager.getLogger(L2AccountManager::class.java)
 )
+
+fun <R, T : Response<R>> retry(
```

Comment:
Can't we use async retryer here instead?

Reply:
I considered that but would imply a even bigger refactor (pass vertx, etc..) and the code is sync. So the cost/benefit did not pay off.